UserBundle/Document/Authorization.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace OpenOrchestra\UserBundle\Document;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Inflector\Inflector;
+use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+use OpenOrchestra\WorkflowFunction\Model\WorkflowFunctionInterface;
+use OpenOrchestra\UserBundle\Model\AuthorizationInterface;
+
+/**
+ * Description of Base Authorization
+ *
+ * @ODM\EmbeddedDocument
+ */
+class Authorization implements AuthorizationInterface
+{
+    /**
+     * @var string $authorizationId
+     *
+     * @ODM\Field(type="string")
+     */
+    protected $authorizationId;
+
+    /**
+     * @var string $name
+     *
+     * @ODM\Field(type="string")
+     */
+    protected $name;
+
+    /**
+     * @var ArrayCollection $workflowFunctions
+     *
+     * @ODM\ReferenceMany(targetDocument="OpenOrchestra\WorkflowFunction\Model\WorkflowFunctionInterface")
```

Comment:
So what should we do? If we just stock strings, each update of workflow will break the authorizations. I add this requirement in composer.json?